prompts/ChatSSH.md

```diff
@@ -0,0 +1,14 @@
+## ChatSSH
+
+
+SSH client for chatting with your Linux Servers
+
+Have more fun and be more productive 
+
+Get work done by asking your Linux SSH Servers to do what you want.
+
+by chatssh.net
+
+https://chat.openai.com/g/g-Q0vUaoZ76-chatssh
+
+```markdown
```

Comment:
The actual prompt seems to be missing.

Reply:
I think the markdown area is for the prompt, at least that's how I notice other GPTs and that's how I shipped this PR:

- https://github.com/linexjlin/GPTs/pull/49